libs/front/components/src/molecules/inputs/input/Input.tsx

```diff
@@ -14,26 +14,20 @@ export function Input<T>(props: InputProps<T>): JSX.Element {
     fieldState,
   } = useController(props);
 
+  const errorMessage = useTranslatedInputError(fieldState);
+
   return (
     <TextField
       label={props.label}
       variant="outlined"
       size="small"
       fullWidth
       error={fieldState.invalid}
-      helperText={t(fieldState.error?.message)}
+      helperText={errorMessage}
```

Comment:
so in hublo-front we currenty have 
```
  const formHelperText = !!fieldState.error
    ? fieldState.error?.message
    : textFieldProps.helperText
```
(as helperText and error message are already translated)